storage/src/hash/mod.rs

```diff
@@ -0,0 +1,444 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
```

Comment:
@tizoc 
can you please put here some short description at the begining with link https://hackmd.io/DnsQrkXCRLmHA3xzLiFgGw?view#Trees ?

Reply:
Ok, but we will have to update that link later once they publish the actual document, since that one is a temporary one where they were doing the editing.

Reply:
@tizoc 
aha, ok, I thought that it is finally